--- jboss/bin/run.conf	2008-02-13 14:02:15 +0100
+++ jboss/bin/run.conf	2008-02-13 14:03:13 +0100
@@ -14,3 +14,27 @@
 if [ -c /dev/urandom ]; then
     JAVA_OPTS="$JAVA_OPTS -Djava.security.egd=file:/dev/urandom"
 fi
+
+# Eclipse Profiler
+if [ -e "$HOME/profilers/active/active/lib/libpiAgent.so" ]; then
+    JAVA_OPTS="$JAVA_OPTS -XrunpiAgent:server=enabled"
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$HOME/profilers/active/active/lib"
+fi
+
+# JBoss Profiler
+if [ -e "$HOME/profilers/active/active/native/linux/libjbossAgent.so" ]; then
+    JAVA_OPTS="$JAVA_OPTS -agentlib:jbossAgent"
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$HOME/profilers/active/active/native/linux"
+fi
+
+# NetBeans Profiler
+if [ -e "$HOME/profilers/active/active/lib/deployed/jdk15/linux/libprofilerinterface.so" ]; then
+    JAVA_OPTS="$JAVA_OPTS -agentpath:$HOME/profilers/active/active/lib/deployed/jdk15/linux/libprofilerinterface.so=$HOME/profilers/active/active/lib,5140"
+fi
+
+# JProfiler Profiler 
+if [ -e "$HOME/profilers/active/active/bin/linux-x86/libjprofiler.so" ]; then
+    JAVA_OPTS="$JAVA_OPTS -agentlib:jprofilerti=port=8849 -Xbootclasspath/a:$HOME/profilers/active/active/bin/agent.jar"
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$HOME/profilers/active/active/bin/linux-x86"
+fi
+
